Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:18-alpine AS base
+WORKDIR /opt/isomercms-backend
+COPY . .
+RUN npm ci
+EXPOSE "8081"
+COPY . /opt/isomercms-backend
+CMD ["npm", "run", "dev:server"]
```

Comment:
how are env vars being injected in this?

Reply:
docker files don't actually do this - this is done inside `docker-compose.yml` through `env-file` and `environment`